apps/ios/Shared/Model/SimpleXAPI.swift

```diff
@@ -844,7 +844,7 @@ func apiRejectContactRequest(contactReqId: Int64) async throws {
     throw r
 }
 
-func apiChatRead(type: ChatType, id: Int64, itemRange: (Int64, Int64)) async throws {
+func apiChatRead(type: ChatType, id: Int64, itemRange: (Int64, Int64)? = nil) async throws {
```

Comment:
Don't think it's a good idea to use this API, we don't use it.

There is minUnreadItemId in ChatStats, and the last one is known from preview, so we can use them.